server/app/models/ckb_transaction.rb

```diff
@@ -11,6 +11,14 @@ class CkbTransaction < ApplicationRecord
   validates :transaction_fee, numericality: { greater_than_or_equal_to: 0 }
 
   attribute :tx_hash, :ckb_hash
+
+  def self.find_ckb_transaction(tx_hash)
+    raise Api::V1::Exceptions::CkbTransactionTxHashInvalidError unless (tx_hash.start_with?(ENV["DEFAULT_HASH_PREFIX"]) && tx_hash.length == ENV["DEFAULT_WITH_PREFIX_HASH_LENGTH"].to_i)
```

Comment:
Raise api exception in model has a little strange

Reply:
here you can treat it as just a common error.

Reply:
if it's a common error, move out from `Api::V1`

Reply:
why? this repo is api only, I think I can raise api error here.

Reply:
Raise api error in controller, not model

Reply:
Thank you, I thought about it, what you said makes sense. I will use a better way.